<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatesEditDialog</class>
 <widget class="QDialog" name="DatesEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QDateEdit" name="dateInEdit"/>
   </item>
   <item row="3" column="4" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="CalendarWidget" name="calendarWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date In:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="RoomDataList" name="orderList"/>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateOutEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add Date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove Date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh Calendar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date Out :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="RoomOrderPropertiesWidget" name="propertiesWidget">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="propertiesButton">
     <property name="text">
      <string>Edit Order</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>Controls/calendarwidget.h</header>
  </customwidget>
  <customwidget>
   <class>RoomDataList</class>
   <extends>QListWidget</extends>
   <header>Controls/roomdatalist.h</header>
  </customwidget>
  <customwidget>
   <class>RoomOrderPropertiesWidget</class>
   <extends>QTreeWidget</extends>
   <header>Controls/roomdatalist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatesEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatesEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
